test/synchronicity_test.py

```diff
@@ -478,17 +478,34 @@ async def aclose(self):
 
 def test_generic_baseclass():
     T = typing.TypeVar("T")
+    V = typing.TypeVar("V")
 
-    class GenericClass(typing.Generic[T]):
+    class GenericClass(typing.Generic[T, V]):
         async def do_something(self):
             return 1
 
-    s = synchronicity.Synchronizer(multiwrap_warning=False)
+    s = synchronicity.Synchronizer()
     WrappedGenericClass = s.create_blocking(GenericClass, name="BlockingGenericClass")
-    instance: WrappedGenericClass[str] = WrappedGenericClass()  #  should be allowed
+
+    assert WrappedGenericClass[str, float].__args__ == (str, float)
+
+    instance: WrappedGenericClass[str, float] = WrappedGenericClass()  #  should be allowed
     assert isinstance(instance, WrappedGenericClass)
     assert instance.do_something() == 1
 
+    Q = typing.TypeVar("O")
```

Comment:
Not sure if it matters but the variable name and argument don't match.

Reply:
🤦 I think it's mostly something static type checkers and/or linters would care about, but nice catch, fixing!